ethereum/datastructures/src/testFixtures/java/tech/pegasys/teku/datastructures/util/DataStructureUtil.java

```diff
@@ -381,8 +386,11 @@ public BeaconBlockBody randomBeaconBlockBody() {
         randomSSZList(
             ProposerSlashing.class, Constants.MAX_PROPOSER_SLASHINGS, this::randomProposerSlashing),
         randomSSZList(
-            AttesterSlashing.class, Constants.MAX_ATTESTER_SLASHINGS, this::randomAttesterSlashing),
-        randomSSZList(Attestation.class, Constants.MAX_ATTESTATIONS, this::randomAttestation),
+            AttesterSlashing.class,
+            Constants.MAX_ATTESTER_SLASHINGS,
+            this::randomAttesterSlashing,
+            1),
+        randomSSZList(Attestation.class, Constants.MAX_ATTESTATIONS, this::randomAttestation, 1),
```

Comment:
I'd probably have at least 3 attestations in the blocks by default.  Good to have a slashing added in there that I'm guessing wound up with none before though.

Reply:
Yep exactly the case.